x-pack/legacy/plugins/canvas/canvas_plugin_src/expression_types/embeddable_types.ts

```diff
@@ -6,7 +6,7 @@
 
 // @ts-ignore
 import { MAP_SAVED_OBJECT_TYPE } from '../../../maps/common/constants';
-import { VISUALIZE_EMBEDDABLE_TYPE } from '../../../../../../src/legacy/core_plugins/kibana/public/visualize/embeddable';
+import { VISUALIZE_EMBEDDABLE_TYPE } from '../../../../../../src/legacy/core_plugins/kibana/public/visualize/embeddable/constants';
```

Comment:
@elastic/kibana-canvas I had to change this import, to ensure, that just this const is imported, and not all of centralized dependencies from Visualize plugin.